sdk/search/azure-search/src/main/java/com/azure/search/models/IndexerExecutionStatus.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Defines values for IndexerExecutionStatus.
+ */
+public enum IndexerExecutionStatus {
```

Comment:
Technically missing 'persistentFailure' but we don't use that value anywhere in the backend so probably fine to not include it.

Reply:
@careyjmac If it isn't part of the public API, we shouldn't document it. I thought it was though.

Reply:
It technically is: AzureSearch\Source\Search\Product\SearchCore\Models\2019-05-06\IndexerExecutionStatus.cs

However I know for a fact that there is nowhere that we actually ever set this to persistentFailure in the backend, hence why I believe it wasn't made "public" even though it is technically a public option.

Reply:
@Careyjmac Please ensure it is removed from the documentation and API models then. Looks like no Swagger or SDK changes required.